renovate.json

```diff
@@ -42,6 +42,18 @@
     {
       "matchManagers": ["github-actions"],
       "groupName": "Pipeline dependencies"
+    },
+    {
+      "matchManagers": ["nuget"],
+      "includePaths": [
+        "Workleap.Extensions.Mongo.Analyzers.Tests"
+      ],
+      "matchPackagePatterns": [
+        "^xunit"
+      ],
+      "groupName": "XUnit (Roslyn Analyzer Tests)",
+      "description": "Newer XUnit versions have bugs that prevent the tests roslyn analyzer tests from running",
```

Comment:
Technically it prevent the Roslyn analyzers testing engine to print detailed errors when diagnostic assertions are not right

Reply:
Indeed